index.html

```diff
@@ -0,0 +1,134 @@
+                        <p class="program-desc">Appreciate various creations applying CC license of artists, organized from Center.</p>
+                    </div>
+                    <div class="program-item">
+                        <img class="program-img" src="./images/forum.png">
+                        <p class="program-name">Forum</p>
+                        <p class="program-desc">Have the time to share your thoughts and opinions with experts for each topic.</p>
+                    </div>
+                    <div class="program-item">
+                        <img class="program-img" src="./images/workshop.png">
+                        <p class="program-name">Workshop</p>
+                        <p class="program-desc">Try creating your own work using open source license rather than just watching.</p>
+                    </div>
+                    <div class="program-item">
+                        <img class="program-img" src="./images/ignite.png">
+                        <p class="program-name">CC Ignite</p>
+                        <p class="program-desc">Get opportunities to network with CC affiliates around the world, also after the summit.</p>
+                    </div>
+                </div>
+                <a href="#" class="join-summit">Join CC Global Summit 2015</a>
+                <a href="#" class="see-whole"><u>SEE THE WHOLE PROGRAM</u></a>
+            </div>
+
+            <div class="speakers">
+                <h2>Featured Speakers</h2>
+                <hr class="speakers-header-hr">
+                <div class="speakers-detail">
+                    <div class="speaker speaker-flex">
+                        <div class="speaker-det">
+                            <div class="speaker-img"><img></div>
+                            <div>
+                                <h3 class="speaker-name"></h3>
+                                <p class="speaker-designation"></p>
+                                <hr class="speaker-hr"><br>
+                                <p class="speaker-detail"></p>
+                            </div>
+                        </div>
+                        <div class="speaker-det">
+                            <div class="speaker-img"><img></div>
+                            <div>
+                                <h3 class="speaker-name"></h3>
+                                <p class="speaker-designation"></p>
+                                <hr class="speaker-hr"><br>
+                                <p class="speaker-detail"></p>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+                <a href="#" class="more">MORE <i class="fa-solid fa-angle-down"></i></a>
+                <div class="speaker-desktop">
+                </div>
+            </div>
+        </main>
+
+        <footer>
+            <div class="partners2"><img src="./images/footer.jpg"></div>
+        </footer>
```

Comment:
- [ ] :thinking: Kindly make sure to create the footer section with code instead of a screenshot of Cindy's design to make your project professional :recycle:

Reply:
Created the footer section with code as per instructions.